README.md

```diff
@@ -254,7 +254,7 @@
 
 # Contact
 
-Copyright (c) 2016-2018 MoleculerJS
+Copyright (c) 2016-2020 MoleculerJS
```

Comment:
The update must be done at `templates/readme-template.md`. Can you update it please?